src/effects/Effect.h

```diff
@@ -25,24 +25,33 @@ class WaveTrack;
 
 //! Default implementation of EffectUIValidator invokes ValidateUI
 //! and IsGraphicalUI methods of an EffectUIClientInterface
-/*!
+/*
+ Also pops the even handler stack of a window, if given to the contructor
 
  This is a transitional class; it should be eliminated when all effect classes
  define their own associated subclasses of EffectUIValidator, which can hold
  state only for the lifetime of a dialog, so the effect object need not hold it
 */
-class DefaultEffectUIValidator
+class COMPONENTS_API DefaultEffectUIValidator
```

Comment:
`COMPONENTS_API` seems to be wrong here. I would've expect `AUDACITY_API`

Reply:
Yep, it breaks Windows build

Reply:
Oh heck